docs/data-cookbooks/browse-generation.md

```diff
@@ -88,9 +88,11 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
 
   The output from this step will be passed into the 'ProcessingStep', which in this example will utilize a 'fake' processing lambda we provide for testing/as an example in Core, however to use your own data you will need to write a lambda that generates the appropriate CMR metadata file and accepts and returns appropriate task inputs and outputs.
 
-  From that step we will utilize the core task ```MoveGranules``` to move the granules to the target buckets as defined in the collection configuration.  That step will transfer the files to their final storage location and update the payload.
+  From that step we will utilize a core task ```FilesToGranulesStep``` that will transform the processing output event.input list/config.InputGranules into an array of Cumulus [granules](https://github.com/nasa/cumulus/blob/master/packages/api/models/schemas.js) objects.
```

Comment:
The task is called `files-to-granules`, the Lambda is called `FilesToGranules`, the workflow config step is called `FilesToGranulesStep`. I'd like to settle on one type of name to use.

Reply:
Fair.    Let's use FilesToGranules?

Reply:
Fixed in this instance : 1f553268f